packages/compass-assistant/src/compass-assistant-provider.tsx

```diff
@@ -37,22 +40,22 @@ export const AssistantContext = createContext<AssistantContextType | null>(
 );
 
 type AssistantActionsContextType = {
-  interpretExplainPlan: ({
+  interpretExplainPlan?: ({
```

Comment:
I was thinking about making these optional and using the existence of the callbacks to be able to tell if the thing should be enabled or not, but was worried that it kinda disabled some type checking benefits - you can't tell between the feature being enabled/disabled vs a mock/stub just not containing it, for example.

But I was probably overthinking it.

Lower down like in the connection error entry point it indeed makes sense to have just the optional callback rather than a boolean and the optional callback. This high up.. I don't know. It is probably fine.

I don't have strong feelings about it.